web/pdf_find_bar.js

```diff
@@ -55,11 +62,19 @@ class PDFFindBar {
       this.dispatchEvent("");
     });
 
-    this.bar.addEventListener("keydown", e => {
-      switch (e.keyCode) {
+    this.bar.addEventListener("keydown", ({ keyCode, shiftKey, target }) => {
+      switch (keyCode) {
         case 13: // Enter
-          if (e.target === this.findField) {
-            this.dispatchEvent("again", e.shiftKey);
+          if (target === this.findField) {
+            this.dispatchEvent("again", shiftKey);
+          } else {
+            for (const [elem, evtName] of checkedInputs) {
+              if (target === elem) {
+                elem.checked = !elem.checked;
+                this.dispatchEvent(evtName);
+                break;
+              }
+            }
```

Comment:
```suggestion
            if (checkedInputs.has(target)) {
              target.checked = !target.checked;
              this.dispatchEvent(checkedInputs.get(target));
            }
```

Given that `checkedInputs` is a map, we don't need to iterate its entries one by one.